config-store-yaml/README.adoc

```diff
@@ -0,0 +1,74 @@
+= Using Config Store Yaml
```

Comment:
We need to be consistent about documentation. Please make sure everyone is on the same page and if so create the necessary issues get our markdown converted to asciidoc. 

Personally, I don't think we're going to be writing anything complicated enough here to warrant the need for asciidoc, it probably makes more sense over in stargate/docs. Plus markdown will probably be more contributor friendly (and if docs are easier to write then people are more likely to do it).